develop-rector.sh

```diff
@@ -1,20 +1,15 @@
-# This script would run after 'composer install' and 'composer update' commands.
-# It creates relative symlinks that (might seem confusing at first) connect
-# between the new development directory (drupal-rector/) and the default
-# directory for composer packages (vendor/palantirnet/drupal-rector)
+# This script would run before 'composer install' and 'composer update' commands.
+# It clones palantirnet/drupal-rector repo to under the drupal-rector/ directory.
+# Composer is set to use the cloned repo instead of getting drupal-rector from packagist
+# (This is done through 'type: path' under repositories section in composer.json)
 
+# Clone palantirnet/drupal-rector if the drupal-rector/ directory does not exist.
 DIR="drupal-rector/"
-echo "Creating development environment under ${DIR} directory"
-
 if [ ! -d "$DIR" ]; then
-  git clone git@github.com:palantirnet/drupal-rector.git
+  echo "Creating development environment under ${DIR} directory"
+  git clone https://github.com/palantirnet/drupal-rector.git
```

Comment:
Can we use `git clone git@github.com:palantirnet/drupal-rector.git` which uses ssh rather than HTTP?

Reply:
I used HTTPS because the Github Action test cannot use it, and perhaps people that did not set up SSH on their machine.
I'll try creating a script that try SSH first, and if not possible it will clone it with HTTPS.